miraclelinux/meta-debian-extended
# xorg-server: Update from 1.20.4-1+deb10u8 to 1.20.4-1+deb10u9

# Purpose of pull request
This version fixes following CVEs.
- CVE-2023-1393

DLA: DLA 3372-1


# Test
## How to test
1. Build package
2. Check core-image-weston works
Run xeyes of Xorg application on Xorg-server.

## Test result
Build succeeded.
```
$ bitbake xserver-xorg
Checking Debian Release ...
Fetching Debian Sources.xz ...
Parsing Debian Sources.xz ...
Parsing recipes: 100% |##############################################################################################################################################| Time: 0:00:39
Parsing of 1362 .bb files complete (0 cached, 1362 parsed). 2384 targets, 80 skipped, 6 masked, 0 errors.
NOTE: Resolving any missing task queue dependencies

Build Configuration:
BB_VERSION           = "1.42.0"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "ubuntu-20.04"
TARGET_SYS           = "aarch64-emlinux-linux"
MACHINE              = "qemuarm64"
DISTRO               = "emlinux"
DISTRO_VERSION       = "2.7"
TUNE_FEATURES        = "aarch64 armv8a crc"
TARGET_FPU           = ""
meta                 
meta-yocto-bsp       = "HEAD:d4b57c68b22027c2bedff335dee06af963e4f8a8"
meta-debian          = "HEAD:04469d4b27cb5203de34211f3fd8248a510b6c35"
meta-debian-extended = "update-xorg-server_1.20.4-1+deb10u9:8eac77b36d3b2e9c33f6c02bd3d31dc767a1b5b5"
meta-emlinux         = "HEAD:c0c24adb1bdb7f4e479976541314e9d9f1bb1f10"
meta-emlinux-private = "HEAD:b290dafea47dfbd8df7ee9b04af38c9b73c87b68"

NOTE: Fetching uninative binary shim from http://downloads.yoctoproject.org/releases/uninative/2.9/x86_64-nativesdk-libc.tar.xz;sha256sum=d07916b95c419c81541a19c8ef0ed8cbd78ae18437ff28a4c8a60ef40518e423
Initialising tasks: 100% |###########################################################################################################################################| Time: 0:00:01
Sstate summary: Wanted 576 Found 0 Missed 576 Current 0 (0% match, 0% complete)
NOTE: Executing SetScene Tasks
NOTE: Executing RunQueue Tasks
NOTE: Tasks Summary: Attempted 2570 tasks of which 0 didn't need to be rerun and all succeeded.
```

Xorg and xeyes is working.
![Screenshot from 2023-04-14 18-00-32](https://user-images.githubusercontent.com/118246107/232356353-62537a4f-c95b-4e8e-890c-4a9f43640e78.png)
